### ISSUE TITLE
Label encoding

### ISSUE THREAD

**Issue by karlvmsousa** [karlvandesman/IN1102-Machine-Learning#1]
Transform labels (forward and backward) between numerical and categorical data

**Comment by karlvmsousa** [comment]
```
from sklearn.preprocessing import LabelEncoder

# create an array with classes names (and sort it)
classes = np.sort(dataset.index.unique().values)

# create an object to transform the labels
encoding = LabelEncoder()
encoding.classes_ = classes

# forward: numerical -> categorical, encoding.inverse_transform(numericalLabel)
# backward: categorical -> numerical, encoding.transform(categoricalLabel)
```
